actions/gallery_actions.js

```diff
@@ -154,7 +155,7 @@ export const getPhotosFromCameraroll = () => async (dispatch, getState) => {
                     lon: item.node.location.longitude,
                     timestamp: item.node.timestamp,
                     selected: false,
-                    picked_up: false,
+                    presence: user.items_remaining === 0 ? false : true,
```

Comment:
Why are we changing the key to presence? `picked_up` is much easier to understand. We should also update the backend to `picked_up`

Reply:
I have made a [trello card](https://trello.com/c/UPqRuLj5) for it..

currently backend logic expects `presence` i.e if `true` then  litter is still there.
but `pickedUp:true` means litter is NOT there.

we need to change backend also.

But this code is not relevant anymore after #129 .
backend needs to be changed

Reply:
We want to use `picked_up`, not `presense`

Reply:
picked_up not added here anymore.